tests/xtts_tests/test_xtts_v2-0_gpt_train.py

```diff
@@ -0,0 +1,166 @@
+import os
```

Comment:
Might be a good idea to wrap most of this code in a `main()` like the other trainer script introduced in this PR does.

Reply:
Agreed but will keep it as it is for this PR